lib/CL/pocl_llvm_build.cc

```diff
@@ -40,6 +40,10 @@ IGNORE_COMPILER_WARNING("-Wstrict-aliasing")
 #include <clang/Frontend/TextDiagnosticPrinter.h>
 #include "clang/Lex/PreprocessorOptions.h"
 
+#ifdef LLVM_10_0
```

Comment:
Please do not use this macro, unless you really need to `#ifdef` on a specific version. Generally if Clang/LLVM changes something in version X, the change stays in future versions too, so it makes more sense to use "before X / after X" kind of macros. Pocl has `LLVM_OLDER_THAN_X_Y` macros for this purpose.